sound/Scale.java

```diff
@@ -6,14 +6,34 @@
 /**
  * A musical scale.
  * Comprised of an ordered set of pitches (midi note numbers).
- * 
+ *
  * @author Dr. Jody Paul
  * @version 1.2
  */
 public class Scale {
     /** The pitches for this tune. */
     private List<Integer> scalePitches;
-    
+    /** Start of pitchInterval.*/
+    private static final int START_PITCH_INTERVAL = -3;
+    /** End of pitchInterval.*/
+    private static final int END_PITCH_INTERVAL = -3;
```

Comment:
Confusing terms "START_PITCH_INTERVAL" and "END_PITCH_INTERVAL", likewise the associated Javadoc comments.  Also, why are they both the same value "-3"?

Reply:
Since these values represent are seven octave scales I could name these START_OCTAVE and END_OCTAVE. Unfortunately that does not explain why the numbers are -3 - 4 instead of 0-6. It may make more sense to name the number 12 OCTAVE_CONSTANT and the loop limits START_OCTAVE_FACTOR and END_OCTAVE_FACTOR, although those names are pretty long. Do you have any recommendations?

Reply:
Also, I think I will have to close this PR and do one more that references these changes. Unfortunately I cannot seem to merge my origin with teams organization (for this branch).

Reply:
> Do you have any recommendations?

I think this whole discussion is a good indicator that the code was written simply to "work" but not for "maintainability".  It has very low understandability, revealed by how difficult it is to ascribe domain-based meaning to the magic numbers.

What's truly needed is a rewrite of the code to make it more understandable.  Having written it, I can make sense of the intent... use "middle C" as the anchor point and add a few octaves below and above that point.  But the "clever" approach to writing a simple loop, has already cost excessive time trying to figure out what it's doing, how to explain that to someone else, and could be a time-sink for someone asked to change it.

So... I like your proposal.  I'm not worried about symbolic constants being too long.  They are meant to be read and only need to be typed a couple of times.  Since I can't resist making some suggestion, how about having factor constants read as `OCTAVE_START_OFFSET` and `OCTAVE_END_OFFSET`.

Thanks for putting up with this poor quality code and helping to make it better!!